US-CBP/design-system
# Chore/more cleanup

#### What's this PR do?
- Added `addon/a11y` npm package for accessibility testing within Storybook
- Added `vite-plugin-static-copy` to copy folders/files into `/dist`
- Changed theme tokens to match proposed color changes
- Removed unneeded images from `/assets`

#### How should this be manually tested?
- **Run `npm install`**
- Check token values against documented changes, run `npm build-tokens` and check tokens build correctly
- Run `npm run styles-build` and check `/assets` and `/sass` folder is copied to `/dist`
- Check storybook for "Accessibility" tab
